index.md

```diff
@@ -22,6 +22,7 @@
     RandomVectorNodeMK2
     SvBricksNode
     ImageNode
+    SvTorusKnotNode
```

Comment:
Yeah. you're right. I thought about but i wasn't sure. I'll make it ext in the next update once I vectorize it.